package.json

```diff
@@ -21,7 +21,8 @@
     "index.html",
     "scripts/",
     "src/",
-    "test/"
+    "test/",
+    "bin"
```

Comment:
master cannot be released right now as `bin` does not get published.

Reply:
nobody really uses the bin :P

Reply:
yeah it just fails to install